x-pack/plugins/reporting/tsconfig.json

```diff
@@ -20,7 +20,6 @@
     { "path": "../../../src/plugins/ui_actions/tsconfig.json" },
     { "path": "../../../src/plugins/usage_collection/tsconfig.json" },
     { "path": "../../../src/plugins/field_formats/tsconfig.json" },
-    { "path": "../event_log/tsconfig.json" },
```

Comment:
Shouldn't `eventLog` be removed from `kibana.json` as well?

Reply:
75591f3f182